script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
+        print(json.dumps(json.loads(json_file.read()), indent=2))
+
+def action_yaml(args):
+    with open('scripts/example_data.yaml') as yaml_file:
+        print(yaml.dump(yaml.safe_load(yaml_file.read()), indent=2))
+
```

Comment:
Similar to the JSON handling, use `yaml.safe_load(yaml_file)` directly instead of `yaml.safe_load(yaml_file.read())` to avoid reading the entire file into memory.
```suggestion
print(yaml.dump(yaml.safe_load(yaml_file), indent=2))
```